### DISCUSSION TITLE
Customize for documentation-collaborative projects

### DISCUSSION THREAD

**Discussion by JuanDCG314** [mmistakes/minimal-mistakes#3212] [General]
Dear,

I'm new to Jekyll and Ruby...

I would like build 10-20 documentation gh-pages with mmistakes and I would like some advice for it.

What is your better approach to achieve it?

Download zip, customize and build a custom theme?
Use theme with the default configuration?
Fork and customize ?
Remote theme?

Use pages?
Use collections?

Thank you very much for your expert advice.

**Comment by iBug** [comment]
A better approach is to use [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).

**Reply by JuanDCG314** [reply]
Thanks for your right and fast answer. 

I'm going to evaluate it.
